maison-de-l-orient-et-de-la-mediterannee.csl

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="sort-only" default-locale="fr-FR" page-range-format="expanded">
+<info>
+    <title>Maison de l'Orient et de la Méditerranée (French)</title>
+    <id>http://www.zotero.org/styles/maison-de-l-orient-et-de-la-mediterannee</id>
+    <link href="http://www.zotero.org/styles/maison-de-l-orient-et-de-la-mediterannee" rel="self"/>
+    <link href="http://www.zotero.org/styles/collection-de-l-ecole-francaise-de-rome-full-note" rel="template"/>
+    <link href="http://www.mom.fr/sites/mom.fr/files/img/Publications/recommandationsauteurs.pdf" rel="documentation"/>
+    <author>
+      <name>Magali Lugnot</name>
+      <email>magali.lugnot@mom.fr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <issn>1274-6525</issn>
+    <eissn>1955-4982</eissn>
+    <summary>Feuille de style pour les publications de la Maison de l'Orient et de la Méditerranée.</summary>
+    <updated>2016-02-18T19:12:02+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="in">in</term>
+      <term name="anonymous">anonyme</term>
+      <term name="anonymous" form="short">anon.</term>
+      <term name="accessed">consulté en</term>
+      <term name="no date">sans date</term>
+      <term name="translator" form="short">trad.</term>
+      <term name="editor" form="short">
+        <single>éd.</single>
+        <multiple>éd.</multiple>
+      </term>
+      <term name="no date" form="short">(s. d.)</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author" delimiter=", " >
+	<name and="text" name-as-sort-order="all" sort-separator=" " delimiter=", " delimiter-precedes-last="never" initialize-with=".">
+        <name-part name="family" font-variant="small-caps" />
+  
+		<name-part name="given"/>
+      </name>
```

Comment:
@Lugnot, as you can see, the indentation here and in other places is a bit messy. Could you copy and paste your style into http://formatter.citationstyles.org/ to correct the indentation, and update your pull request?